data.go

```diff
@@ -371,18 +371,28 @@ func findScopesForId(configId string) (scopes []string) {
 
 	log.Debugf("findScopesForId: %s", configId)
 
-	var scope string
+	var scope sql.NullString
 	db := getDB()
 
-	rows, err := db.Query("select DISTINCT scope from EDGEX_DATA_SCOPE where apid_cluster_id = $1", configId)
+	query := `
+		SELECT scope FROM edgex_data_scope WHERE apid_cluster_id = $1
```

Comment:
Yes, I agree that UNION ALL is more performant. Which two SELECTs are already producing distinct results?